apis/kots/v1beta1/installation_types.go

```diff
@@ -34,6 +34,7 @@ type InstallationSpec struct {
 	ReplicatedChartNames     []string                `json:"replicatedChartNames,omitempty"`
 	EncryptionKey            string                  `json:"encryptionKey,omitempty"`
 	KnownImages              []InstallationImage     `json:"knownImages,omitempty"`
+	AirgapArtifacts          []string                `json:"airgapArtifacts,omitempty"`
```

Comment:
I would rename `AirgapArtifacts` to `EmbeddedClusterArtifacts` maybe? as it’s pretty specific to embedded clusters.